frontend/src/components/CompanySearch/components/SearchHistory/SearchHistory.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './SearchHistory.css';
+
+export default function() {
+
+    return (
+        <div className="search-history">
+            Здесь будет история поиска
+        </div>
+    );
+}
```

Comment:
у компонентов в реакте должны быть имена, использование безымянных функций делает displayName равным Component и отлаживать такой код будет гораздо сложнее 
```suggestion
const SearchHistory = () => (<div className="search-history">Здесь будет история поиска</div>);
export default SearchHistory;
```